doc/source/user_guide/skimage2_migration.md

```diff
@@ -0,0 +1,30 @@
+(skimage2_migration)=
+
+# Transition to skimage2
+
+:::{hint}
+This document is a work in progress and still subject to change.
+:::
+
+scikit-image is preparing to release version 2.0 under the new namespace `skimage2`.
+This will affect both its import name and its package name on PyPI and elsewhere.
+Together with skimage2, we will release version 1.0.0 and 1.0.1 with the old API.
```

Comment:
```suggestion
Alongside skimage2, we will release version 1.0.0, the last using the current API.
```

Maybe this comes after the explanation of why we are doing it (paragraph below).